book/src/using_rusty/build_description_file.md

```diff
@@ -1,39 +1,46 @@
-# Build description File
+# Build Command 
 
-In addition to the comprehensive help, `rustyc` offers a build description file, that simplifies the build process. Instead of having numerous inline arguments, using the build description file makes passing the arguments easier and neater. The build description file needs to be safed as a [json](https://en.wikipedia.org/wiki/JSON) format.
+In addition to the comprehensive help, `rustyc` offers a build subcommand that simplifies the build process. </br>
+Instead of having numerous inline arguments, using the build subcommand along with a build description file makes passing the arguments easier. </br>
+The build description file needs to be saved in the [json](https://en.wikipedia.org/wiki/JSON) format.
 
-`rustyc build [Config]`
+Usage: 
+
+`rustyc build`
 
 Note that if `rustyc` cannot find the `plc.json` file, it will throw an error and request the path. The default location for the build file is the current directory. The command for building with an additional path looks like this:
 
 `rustyc build src/plc.json`
 
 
-# Plc.json
+## Build description file (plc.json)
 
-For the build description file to work, the build description file must be the [json](https://en.wikipedia.org/wiki/JavaScript_Object_Notation) format. All the keys used in the build description file are described in the following sections. 
+For the build description file to work, it must be written in the [json](https://en.wikipedia.org/wiki/JavaScript_Object_Notation) format. All the keys used in the build description file are described in the following sections. 
 
 
-## files
+### files
 
-The key `files` is the equivalent to the `input` parameter, which adds all the `.st` files that needs to be compiled. The value of `files` is an array of strings, definied as followed:
+The keyword `files` is the equivalent to the `input` parameter, which adds all the `ST` files that need to be compiled. The value of `files` is an array of strings, definied as follows:
 ```json
 "files" : [
     "examples/hello_world.st",
     "examples/hw.st"
+    "examples/*.gvl"
 ]
 ```
 
 
-## libraries
+### libraries
 
-To link several executables `rustyc` has the option to add libraries and automatically build and like them together. if no compile type has been selected `rustyc` will link the files on default.
+To link several executables `rustyc` has the option to add libraries and automatically build and link them together.</br>
```

Comment:
"To link several executables..." is not exactly what were trying to say?
maybe "To link several objects into one executrable...?" idk ...